models/user.js

```diff
@@ -1,10 +1,38 @@
-const mongoose = require("mongoose");
+var mongoose = require("mongoose"),
+  // Schema = mongoose.Schema,
+  bcrypt = require("bcrypt"),
+  SALT_WORK_FACTOR = 10;
 
-const UserSchema = new mongoose.Schema({
-  email: String,
-  password: String,
+var UserSchema = new mongoose.Schema({
```

Comment:
Did you make this var? Should we change this to let?

Reply:
We can, I didn't make it it was already there